code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -68,9 +63,9 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 	///Minimum pressure allower for release_pressure var
 	var/can_min_release_pressure = (ONE_ATMOSPHERE * 0.1)
 	///Max amount of heat allowed inside of the canister before it starts to melt (different tiers have different limits)
-	var/heat_limit = 5000
+	var/heat_limit = 50000
```

Comment:
you wanna keep the pressure high, but heat limit at t1? That way people have more of a reason to use the shield often

Reply:
uh i wanted to increase the heat limit a bit, so i'll lower this to 10k K to give at least a bit of time to react in case of a fast burning canister

Reply:
aight chill